R/execution.r

```diff
@@ -216,13 +238,19 @@ execute = function(request) {
     send_response('execute_input', request, 'iopub', list(
         code = request$content$code,
         execution_count = execution_count))
-        
+
```

Comment:
please configure your editor so it doesn’t touch lines you didn’t touch. this whitespace (and the whitespace in multiple more lines below) should stay.